mipengine/node/monetdb_interface/common.py

```diff
@@ -27,6 +30,7 @@
 cursor = connection.cursor()
 
 
+@validate_identifier_names
 def create_table_name(table_type: str, command_id: str, context_id: str, node_id: str) -> str:
     """
     Creates a table name with the format <table_type>_<context_id>_<node_id>_<uuid>
```

Comment:
this table naming convention is not up to date
<tableType>_<commandId>_<contextId>_<nodeId>